### DISCUSSION TITLE
Math tabular alignment

### DISCUSSION THREAD

**Discussion by Kreijstal** [typst/typst#973] [Q&A] [answered]
how to align equations to equality?
Like this:
https://www.overleaf.com/learn/latex/Aligning_equations_with_amsmath

I looked up `align` in reference, and its just mere text alignment.

**Comment by dvdvgt** [comment, marked as answer]
I don't know if I understand correctly. Is this what you are looking for? https://typst.app/docs/reference/math/

You can align equations with an `&` like this:

```
$
sum_(k=0)^n k &= 1 + ... + n \
&= (n(n+1)) / 2
$ 
```

**Reply by Kreijstal** [reply]
You are completely right, I guess I should use google to search instead?
![grafik](https://user-images.githubusercontent.com/2415206/234241611-c1d14344-421a-4a2e-a6c3-53763ff064e4.png)
Thank you very much

**Reply by LaurenzV** [reply]
Yeah, the search function for the documentation sometimes doesn't work that well, unfortunately.
